sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=public-convenience-ltd
+defaults.project=toilet-map
+cli.executable=../../../.npm/_npx/a8388072043b4cbc/node_modules/@sentry/cli/bin/sentry-cli
```

Comment:
this seems like a super specific path for npx — will this be stable, or should it hit just `node_modules/@sentry/cli/bin/sentry-cli`?

Reply:
The sentry tool generated this. I don't _think_ we even need it...